charts/kube-acp-stack/values.yaml

```diff
@@ -40,3 +40,10 @@ redis-cluster:
     extraVolumeMounts:
       - mountPath: /redismodules
         name: redismodules
+timescaledb-single:
+  enabled: true
+  secrets:
+    credentials:
+      PATRONI_SUPERUSER_PASSWORD: "PaSsW0rD"
```

Comment:
I think we can set only superuser password

Reply:
done.